src/OrchardCore.Build/AspNetCoreVersion.props

```diff
@@ -0,0 +1,7 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
```

Comment:
Curious: have you consider renaming this file and adding all the external dependencies versions? (e.g Fluid, YesSql, OpenIddict, etc.)

That would be a great way to have a centralized place where versions are defined.